src/test/testFixtures/common/collections/ansible_collections/testorg/collection_1/galaxy.yml

```diff
@@ -0,0 +1,9 @@
+---
+authors:
+  - Ansible team (ansible-navigator)
```

Comment:
Minor change. This can be just `Ansible Devtools team`

Reply:
Thanks. Updated !